sopel/config/types.py

```diff
@@ -515,10 +524,17 @@ def parse(self, value):
                 for item in value.splitlines())
         else:
             # this behavior will be:
-            # - Discouraged in Sopel 7.x (in the documentation)
```

Comment:
Removing this seems unnecessary. Why not keep the history around as a reminder to those of us actually maintaining the code? It won't appear in docs.

Reply:
Yeah sure, at some point the whole block will be removed anyway.